2023/README.md

```diff
@@ -78,11 +76,11 @@ show that specific test case among various of testruns, `show [test_run_id];` wi
 
 
 ### 2. Keploy Java SDK
-**Mentors**: Shubham Jain, Sarthak Shyngle, Gourav Kumar.
+**Mentors**: Sarthak Shyngle, Gourav Kumar, Charan Kamarapu.
 
 **Goals & Ideas**
-* Add support for `MongoDb` framework.
-* 
+* Add support for `MongoDB` framework.
+* Add support for `Apache Kafka`.
```

Comment:
We need more frameworks to add as a task. Maybe let's add some more popular ones.

Reply:
We can add RabbitMQ, GCP, and Azure SDKs. Native HTTP Client